### PR TITLE
Control plane adoption rollback

### PR DESCRIPTION
**Pull request by jistr**

### CODE REVIEW

File: tests/roles/stop_openstack_services/tasks/main.yaml [hunk 1]
```diff
@@ -57,9 +66,10 @@
         for i in {1..3}; do
             SSH_CMD=CONTROLLER${i}_SSH
             if [ ! -z "${!SSH_CMD}" ]; then
-                echo "Checking status of $service in controller $i"
                 if ! ${!SSH_CMD} systemctl show $service | grep ActiveState=inactive >/dev/null; then
                     echo "ERROR: Service $service still running on controller $i"
+                else
+                    echo "OK: Service $service is not running on controller $i"
```
Comment (fao89) [inline tests/roles/stop_openstack_services/tasks/main.yaml:72]: 👍🏽

**Review by fao89 — APPROVED** [review]

**Comment by jistr** [comment]
recheck

**Comment by jistr** [comment]
recheck

**Comment by fao89** [comment]
recheck

File: docs_user/modules/openstack-rolling_back.adoc [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                "tripleo_cinder_backup.service"
+                "tripleo_glance_api.service"
+                "tripleo_manila_api.service"
+                "tripleo_manila_api_cron.service"
+                "tripleo_manila_scheduler.service"
+                "tripleo_neutron_api.service"
+                "tripleo_nova_api.service"
+                "tripleo_placement_api.service"
+                "tripleo_nova_api_cron.service"
+                "tripleo_nova_api.service"
```
Comment (archana203) [inline docs_user/modules/openstack-rolling_back.adoc:54]: nova_api.service is duplicated in the list.

Comment (jistr) [reply docs_user/modules/openstack-rolling_back.adoc:54]: Thanks, i added a commit to fix service lists into a WIP branch here https://github.com/jistr/data-plane-adoption/tree/f-rollback-tests

**Pull request merged**